test/Serilog.Sinks.NLog.Tests/project.json

```diff
@@ -2,7 +2,9 @@
   "testRunner": "xunit",
   "dependencies": {
     "xunit": "2.2.0-beta2-build3300",
+    "xunit.runner.visualstudio": "2.1.0",
     "dotnet-test-xunit": "2.2.0-preview2-build1029",
+    "NLog": "5.0.0-beta03",
```

Comment:
My bad, I didn't pay attention when adding the reference; it should obviously be the same as the one in the main project 👍